src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  cursor: pointer;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  @for $row from 1 through 4 {
+    &--#{$row} {
```

Comment:
do it with nth-child 
Check it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)